pulp_ansible/app/models.py

```diff
@@ -10,30 +10,14 @@
 
 class AnsibleRole(Content):
     """
-    A model representing an Ansible Role.
+    A content type representing an Ansible Role version.
     """
 
     TYPE = 'ansible-role'
```

Comment:
This causes the API serializer to have `"_type": "ansible.ansible-role",` What about renaming `AnsibleRole` to `Role` throughout and this to `'role'`?

Reply:
We can handle this separately. I wrote this issue and put it onto the 0.1 release. https://pulp.plan.io/issues/4709